src/main/java/herrekt/TaskList.java

```diff
@@ -0,0 +1,63 @@
+package herrekt;
+
+import herrekt.taskmanager.Task;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class TaskList {
+    private List<Task> tasks;
+
+    TaskList(List<Task> tasks) {
+        this.tasks = tasks;
+
```

Comment:
Remove extra blank lines like this that are not consistent with other method layout in the file.